scout/server/static/bs_styles.css

```diff
@@ -49,6 +49,44 @@ a.dropdown-item:visited {
     background-color: var(--bs-info);
 }
 
+/* define colors to be used as class items */
+.bg-red {
+ background-color: var(--bs-red);
+}
+
+.bg-blue {
+ background-color: var(--bs-blue);
+}
+
+.bg-teal {
+ background-color: var(--bs-teal);
+}
+
+.bg-green {
+ background-color:: var(--bs-green);
```

Comment:
```suggestion
 background-color: var(--bs-green);
```

Reply:
Dunno; seems to work either way, so most parsimonious is single colon. 🤷

Reply:
ah yes, a typo. Thanks!